react-client/src/Components/App.js

```diff
@@ -63,6 +66,7 @@ function App() {
             id: e.target.id,
             clsname: e.target.className,
             children: children,
+            childcount: e.target.childElementCount,
```

Comment:
Redundant. Please remove

Reply:
noted already removed made a modification sa state from app.js to dialog box component